package.json

```diff
@@ -14,56 +16,65 @@
         "not dead"
     ],
     "dependencies": {
-        "@babel/core": "7.0.0-beta.46",
+        "@babel/core": "^7.2.2",
         "@babel/plugin-proposal-object-rest-spread": "7.0.0-beta.46",
```

Comment:
Most plugins have been updated to the non-beta version. Perhaps we should do the same here?

Currently at `7.4.0`.